app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsEffectHandler.kt

```diff
@@ -43,4 +49,21 @@ class PatientSearchResultsEffectHandler @AssistedInject constructor(
           }
     }
   }
+
+  private fun openPatientEntryScreen(): ObservableTransformer<OpenPatientEntryScreen, PatientSearchResultsEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulers.io())
+          .switchMap {
+            userSession
+                .loggedInUser()
```

Comment:
We only need `Facility` in `openPatientEntryScreen`, right? Why do we need to add loggedInUser here?

Reply:
`FacilityRepository#currentFacility` requires the current user a parameter to fetch the current facility.